app/tasks/raster_tile_set_assets/utils.py

```diff
@@ -141,13 +141,18 @@ async def create_gdaldem_job(
         target_prefix,
     ]
 
+    kwargs = dict()
```

Comment:
minor, but is there a way to not use kwargs for this? I think we overuse submitting params as kwargs sometimes, and it gets confusing what's being used where. I think we could just set a default attempt_duration_seconds if not specified (or leave it None)

Reply:
Hmm. There IS a default attempt_duration_seconds, but it's specified in the model. The problem with passing in a None is that it then won't take the default value (as specified in the Job model). It will wind-up as REALLY None, when we want it to be the default if unspecified:
In [9]: class Blah(BaseModel):
   ...:     foo: Optional[int] = 5
In [10]: some_blah = Blah(foo=None)
In [11]: type(some_blah.foo)
Out[11]: NoneType
There IS a default

Reply:
One option would be to move all the params to a dict called job_params (or whatever), and then just do `return GDALDEMJob((**job_params)`, which might be a bit more clear than mixing **kwargs with named arguements?  That way all parameters are defined together, some of them conditional, before being passed all together to the constructor.

Reply:
Mmm, that's an idea. How do you feel about that Justin?